integration_tests/e2e/suggestions.cy.ts

```diff
@@ -120,6 +120,23 @@ context('Suggestions', () => {
     searchPage.firstName().should('have.value', 'Smith')
   })
 
+  it('Will suggest adding swapping forename and surname when one is empty', () => {
+    cy.task('stubPrisonerSearchByName')
+    const searchPage = Page.verifyOnPage(SearchPage)
+    searchPage.lastName().type('Smith')
+    searchPage.searchButton().click()
+    searchPage.suggestions().click()
+
+    const suggestionsPage = Page.verifyOnPage(SuggestionsPage)
+    suggestionsPage.swap().within(() => {
+      cy.get('span').should('have.text', 'Smith ')
```

Comment:
@petergphillips Not sure if this is what we want (the extra space) but it doesn't matter (I could've used contains text)

Reply:
I wasn't that bothered to remove it since it will be ignored by the browser